bundles/org.eclipse.osgi/META-INF/services/org.osgi.framework.connect.ConnectFrameworkFactory

```diff
@@ -1 +1 @@
-org.eclipse.osgi.launch.EquinoxFactory
+org.eclipse.osgi.internal.framework.ConnectFrameworkUtilHelper
```

Comment:
Did you mean to put that in `META-INF/services/org.osgi.framework.connect.FrameworkUtilHelper` instead?

Reply:
Yes.. I fixed that!